rbac/management/group/definer.py

```diff
@@ -63,12 +65,33 @@ def seed_group(tenant):
     return tenant
 
 
-def set_system_flag_post_update(group):
+def set_system_flag_post_update(group, tenant):
     """Update system flag on default groups."""
     if group.system:
         group.name = "Custom default access"
-    group.system = False
-    group.save()
+        group.system = False
+        group.save()
+        clone_default_group_in_public_schema(group, tenant)
+
+
+def clone_default_group_in_public_schema(group, tenant):
+    """Clone the default group for a tenant into the public schema."""
+    tenant_default_policy = group.policies.get(system=True)
+    public_tenant = Tenant.objects.get(schema_name="public")
+    clear_pk(group)
+    clear_pk(tenant_default_policy)
+    with tenant_context(public_tenant):
+        if Group.objects.filter(name=group.name, platform_default=group.platform_default, tenant=tenant):
+            return
+
+        public_default_roles = (
+            Group.objects.get(platform_default=True, tenant=public_tenant).policies.get(system=True).roles.all()
+        )
+
+        group.save()
+        tenant_default_policy.group = group
```

Comment:
Overall LGTM, probably not a big deal but should we concern about the name of the policy. The name is sth like "Default policy for group xxxxxx", where the xxxxxx is the group uuid within the tenant, and changed after copying into public schema.

Reply:
Chatted with Keith, this won't be a problem since the UUID is not changed after clear PK